src/Features/VisualBasic/Portable/Completion/SuggestionMode/VisualBasicSuggestionModeCompletionProvider.vb

```diff
@@ -97,11 +97,45 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Completion.SuggestionMode
             If syntaxTree.IsNamespaceDeclarationNameContext(position, cancellationToken) Then
 
                 description = VBFeaturesResources.TypeANameHereToDeclareANamespace & vbCrLf &
-                              VBFeaturesResources.NoteSpaceAndCompletion
+                              VBFeaturesResources.NoteSpaceCompletionIsDisa
```

Comment:
Yeah, what's up with that? ;-) Both of these are existing resources. As much as I want to fix it, I suspect changing the name might interfere with the loc process.

As for the change itself, the new one has better wording - it doesn't mention anything about committing on `=`.